sklearn/linear_model/logistic.py

```diff
@@ -1315,7 +1315,7 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
             Default will change from 'liblinear' to 'lbfgs' in 0.22.
 
     max_iter : int, optional (default=100)
-        Useful only for the newton-cg, sag and lbfgs solvers.
+        Useful only for the newton-cg, sag, lbfgs, liblinear and saga solvers.
```

Comment:
I guess we should simply remove this line, because this list includes all supported solvers.

Reply:
+1
I would just write:

`The maximum number of iterations used by the solver.`

Reply:
+1

Reply:
Makes sense, simplification incoming...